### ISSUE TITLE
OpenBSD version

### ISSUE THREAD

**Issue by freeman1doma** [erebe/wstunnel#377]
## Describe the feature
Do you planning to make OpenBSD version of wstunnel?

**Comment by erebe** [comment]
Hello,

There is no rust toolchain (compiler and co) available for openbsd, at least available for cross compilation. So at the moment it is not possible to do a build for it.

If you managed to have an openbsd available try to install rust on it, and you should be able to compile the project `cargo build --release`

Let me know

**Comment by Sir-Photch** [comment]
There is a WIP port on the mailing list that is waiting for another OK:
https://marc.info/?l=openbsd-ports&m=173497762523884&w=2

**Comment by Sir-Photch** [comment]
Yes, I have noticed. But right now, it does not matter. As I see it, the port is complete and I have it working properly on my setup. That patch is just a little chore. It will probably be removed when you release the next version. There is no need for you to rush another release.

But if you'd really like to have good OpenBSD compatibility, you can consider setting your MSRV to that of OpenBSD-current (which is the development "branch"): https://openbsd.app/path/lang/rust?current=on
It is 1.83 right now, so there shouldn't be an issue. Though, when a new OpenBSD release is tagged, the rust version freezes, and as there is no rustup on OpenBSD, wstunnel can only be built against the frozen rust version.

**Comment by erebe** [comment]
Understood the MSRV at 1.83.0. Thanks for the feedback

**Comment by shtrophic** [comment]
This can be closed as the port was im-port-ed! 👍🏼
